### PR TITLE
DataSketches aggregator

### PR DESCRIPTION
**Pull request by asaitov**
Added datasketches extension aggregator
http://druid.io/docs/latest/development/extensions-core/datasketches-aggregators.html

### CODE REVIEW

**Comment by bjgbeelen** [comment]
Hi @asaitov,
We're welcoming your MR, but we just did a major change to the Druid API, therefore there are conflicts with your code. If you are able to resolve the conflict, we'll merge your PR right away

**Comment by asaitov** [comment]
Hey @bjgbeelen, please review the changes. Thanks.

**Comment by asaitov** [comment]
@Fokko Done.

File: src/main/scala/ing/wbaa/druid/DruidQuery.scala [hunk 1]
```diff
@@ -30,7 +32,7 @@ object QueryType extends EnumCodec[QueryType] {
   case object TopN       extends QueryType
   case object GroupBy    extends QueryType
   case object Timeseries extends QueryType
-  val values = Set(TopN, GroupBy, Timeseries)
+  val values: Set[QueryType] = sealerate.values[QueryType]
```
Comment (bjgbeelen) [inline src/main/scala/ing/wbaa/druid/DruidQuery.scala]: Nice improvement, didn't know about its existence!

**Pull request merged**